### PR TITLE
feat (DOCS-CODE-035): [authN] [sign-in] canonical X-Plat identity sample for user sign-in using in .NET 6 MAUI Part 4

### PR DESCRIPTION
**Pull request by ferantivero**
add details about mobile
remove iOS

### CODE REVIEW

File: src/sign-in-xplat/README.md [hunk 1]
```diff
@@ -10,26 +10,27 @@ products:
 - azure
 - azure-active-directory
 - ms-graph
```
Comment (ckittel) [inline src/sign-in-xplat/README.md:12]: I think that's the right product name.  If it's not this, it's `dotnet-maui` -- I get "products" and "technologies" mixed up in the metadata. Maybe we just wait on this for content development.

```suggestion
- ms-graph
- dotnet-mobile
```

**Review by ckittel — CHANGES_REQUESTED** [review]
Only a couple of quick hits.  Whaddya think?

File: src/sign-in-xplat/XPlat.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
+		<TargetFrameworks>net6.0-android;net6.0-maccatalyst</TargetFrameworks>
```
Comment (ckittel) [inline src/sign-in-xplat/XPlat.csproj:4]: it was both i0S and mac we needed removed.

Comment (ferantivero) [reply src/sign-in-xplat/XPlat.csproj]: you got it @ckittel 

done | addressed from [f1080c](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/18/commits/f1080c85bd3ae08ea847763f6de72aeb201f1760)

**Review by ckittel — APPROVED** [review]
🚀

**Pull request merged**